new_dawn_server/users/api/tests/test_users_api.py

```diff
@@ -34,6 +38,26 @@ def setUp(self):
             "school": "NYU",
             "smoke": "Socially",
         }
+        image1 = SimpleUploadedFile("media/images/test.png", b"file_content", content_type="image/png")
+        image2 = SimpleUploadedFile("media/images/testcat.JPG", b"file_content", content_type="image/jpg")
+        self.image_arguments = {
+            "images": [
+                {
+                    "caption": "no",
+                    "media": image1,
```

Comment:
The format of the image sent from iOS wouldn't be a Django SimpleUploadedFile instance. So this test cannot mimic the real request. Usually you should test a serialized multipart request sent from front-end to make sure backend can handle encoded image and its metainfo in the correct way. You should take a look at the api test in medias/tests directory, which tested remote image upload request without using local uploaded file.